tests/test_oracle/test_oracledb.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+
+import oracledb
+import pytest
+
+from pydapper import connect
+from pydapper import using
+from pydapper.oracle import OracledbCommands
+from tests.suites.commands import ExecuteScalarTestSuite
+from tests.suites.commands import ExecuteTestSuite
+from tests.suites.commands import QueryFirstOrDefaultTestSuite
+from tests.suites.commands import QueryFirstTestSuite
+from tests.suites.commands import QueryMultipleTestSuite
+from tests.suites.commands import QuerySingleOrDefaultTestSuite
+from tests.suites.commands import QuerySingleTestSuite
+from tests.suites.commands import QueryTestSuite
+
+
+@pytest.fixture(scope="function")
+def commands(server, database_name) -> OracledbCommands:
+    with OracledbCommands(
+        oracledb.connect(password="pydapper", user="pydapper", dsn=f"{server}:1522/{database_name}")
+    ) as commands:
+        yield commands
+
+
+def test_using(server, database_name):
+    with using(
+        oracledb.connect(password="pydapper", user="pydapper", dsn=f"{server}:1522/{database_name}")
+    ) as commands:
+        assert isinstance(commands, OracledbCommands)
+
+
+@pytest.mark.parametrize("driver", ["oracle", "oracle+oracledb"])
+def test_connect(driver, database_name, server):
```

Comment:
you can remove the parameterize, I think I'm going to leave cx oracle as the oracle default for now until I can confirm that doesn't affect others

Reply:
Do you mean I should delete this test entirely?

Reply:
I just changed it to test the usage of the `oracle+oracledb` DSN. Let me know if that works!

Reply:
thats perf, ty